### PR TITLE
Creating README.md.

### PR DESCRIPTION
**Pull request by davidyuyuan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# covid-sequence-analysis-workflow
+
+This is the official repository of the SARS-CoV-2 variant surveillance pipeline developed under the [Versatile Emerging infectious disease Observatory (VEO)](https://www.globalsurveillance.eu/projects/veo-versatile-emerging-infectious-disease-observatory) project. The project consists of 20 European partners. It is funded by the European Commission.
```
Comment (nadimm-rahman) [inline README.md]: Maybe mentioning EMC, ELTE, DTU and EMBL-EBI outright would be good.

'...developed by Danish Technical University (DTU), Eotvos Lorand University (ELTE), EMBL-EBI, Erasmus Medical Center (EMC), under the Versatile...'

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
```
Comment (nadimm-rahman) [inline README.md:6]: Maybe can also mention that the pipeline has been integrated on EMBL-EBI infrastructure to automatically process raw SARS-CoV-2 read data, presenting in the COVID-19 Data Portal: https://www.covid19dataportal.org/sequences?db=sra-analysis-covid19&size=15&crossReferencesOption=all#search-content.

**Review by nadimm-rahman — COMMENTED** [review]
Hi!
Overall this is very helpful and clear, I left a couple of comments.

Was going to also ask about dependencies, but I think you use Docker images/containers so the user wouldn't have to worry about these.

Many thanks!
Nadim

**Comment by davidyuyuan** [comment]
Thank you, @nadimm-rahman , for the feedback. I have updated top section accordingly.
@carlacummins, any comments?

**Review by carlacummins — APPROVED** [review]

**Pull request merged**